Extension/ui/settings.html

```diff
@@ -355,11 +318,11 @@
   <div id="main" class="main">
 
     <!-- title -->
-    <div style=" height: 90px; display: block">
+    <div style=" height: 90px; display: inline-block;">
       <div class="main-title">IntelliSense Configurations</div>
-      <div>
-        Use this editor to edit basic IntelliSense settings for the active configuration.
-        To edit multiple IntelliSense configurations and additional settings go to <a href="command:C_Cpp.ConfigurationEditJSON" title="Edit configurations in JSON file">c_cpp_properties.json</a>.
+      <div style="color: var(--vscode-foreground);">
+        Use this editor to edit basic IntelliSense settings defined in the underlying <a href="command:C_Cpp.ConfigurationEditJSON" title="Edit configurations in JSON file">c_cpp_properties.json</a> file.
+        Only the setting values of the active configuration can be modified at a time. Switch active configurations to modify a different configurations on this editor. To edit multiple configurations or set additional settings go to <a href="command:C_Cpp.ConfigurationEditJSON" title="Edit configurations in JSON file">c_cpp_properties.json</a>.

```

Comment:
There are a few grammar issues here.  Consider rewording:

`"Changes made in this editor only apply to the active configuration. To modify other configurations <a href="...">change the active configuration</a>. To edit multiple configurations at once, or advanced settings not shown here, go to <a href="...">c_cpp_properties.json</a>"`